bom/operaton-bom/pom.xml

```diff
@@ -47,6 +47,6 @@
   </licenses>
   <organization>
     <name>Operaton</name>
-    <url>http://www.operaton.org</url>
+  <url>https://www.operaton.org</url>
```

Comment:
Please do not change indentations. Also check all other changes, there are also other changes of that kind in the PR.